From f8817b439433798bd7217dc4ae72197887b0fc96 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Thu, 30 Sep 2021 11:12:27 +0200
Subject: [PATCH] cpu: fix compilation on ARM

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[james.hilliard1@gmail.com: backport from upstream commit
f8817b439433798bd7217dc4ae72197887b0fc96]
---
 spa/plugins/support/cpu-arm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/spa/plugins/support/cpu-arm.c b/spa/plugins/support/cpu-arm.c
index d80b8ad87..de07ebf41 100644
--- a/spa/plugins/support/cpu-arm.c
+++ b/spa/plugins/support/cpu-arm.c
@@ -80,7 +80,7 @@ arm_init(struct impl *impl)
 	int arch;
 
 	if (!(cpuinfo = get_cpuinfo())) {
-		spa_log_warn(impl->log, NAME " %p: Can't read cpuinfo", impl);
+		spa_log_warn(impl->log, "%p: Can't read cpuinfo", impl);
 		return 1;
 	}
 
-- 
2.25.1

